app/src/main/java/org/wikipedia/language/LangLinksActivity.kt

```diff
@@ -242,23 +169,27 @@ class LangLinksActivity : BaseActivity() {
         }
 
         override fun getItemCount(): Int {
-            return this.languageEntries.size
+            return languageEntries.size
         }
 
         override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): DefaultViewHolder {
             val context = parent.context
             val inflater = LayoutInflater.from(context)
             return if (viewType == Companion.VIEW_TYPE_HEADER) {
                 val view = inflater.inflate(R.layout.view_section_header, parent, false)
-                DefaultViewHolder(languageEntries, view)
+                DefaultViewHolder(view)
             } else {
                 val view = inflater.inflate(R.layout.item_langlinks_list_entry, parent, false)
-                LangLinksItemViewHolder(languageEntries, view)
+                LangLinksItemViewHolder(view)
             }
         }
 
         override fun onBindViewHolder(holder: DefaultViewHolder, pos: Int) {
-            holder.bindItem(pos)
+            if (variantTitlesToUpdate.contains(languageEntries[pos])) {
+                viewModel.fetchLangVariantLink(languageEntries[pos])
+                variantTitlesToUpdate.remove(languageEntries[pos])
+            }
```

Comment:
Looks like there's a scrolling issue when language variants are added, not sure they are related.

Steps to reproduce:
1. Search "Island groups of the Philippines" 
2. Change language to "Traditional Chinese" (Please make sure there's only one `zh` app language)
3. Click on "Language" again and scroll down to the bottom of the list.
   
https://www.youtube.com/watch?v=IGJ_EzhZRVE

Reply:
It doesn't seem like a very significant issue (and I can't reproduce it on my device). It's probably because I'm using `notifyItemRangeChanged()`, which is more efficient but produces an animation, instead of `notifyDataSetChanged()`. If it's not too distracting, and if all the text is correct, can we keep it this way?

Reply:
Got it. If it is possible, can we apply `binding.langlinksRecycler.itemAnimator = null` to the `RecyclerView` so that people will not see the variant titles pop up?

Reply:
yep!